custom-connectors/PUGGamifiedEngagement/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "INTEGRATION API KEY",
```

Comment:
@smart-jp Please let us know why you want to show the API Key field name in all capital letters.